apps/challenges/aws_utils.py

```diff
@@ -1025,6 +1023,12 @@ def restart_workers_signal_callback(sender, instance, field_name, **kwargs):
                     )
 
 
+post_save_connect("evaluation_script", Challenge,
```

Comment:
@burnerlee moving post save callback register logic to `aws_utils` doesn't make sense. We need a cleaner way to fix the cyclic imports. It's fine if we have to refactor some part of the code but let's keep callbacks in `models.py`.